### PR TITLE
Fix sign in on iOS8

### PR DESCRIPTION
**Pull request by walaura**
we had a bit of polyfill troubles. it's all good now.

### CODE REVIEW

File: .browserslistrc [hunk 1]
```diff
@@ -1 +1,3 @@
 IE >= 9
+iOS >= 8
```
Comment (rahilb) [inline .browserslistrc]: Does this mean iOS versions 8 or more will be polyfilled?

Comment (walaura) [reply .browserslistrc]: yep

File: package-lock.json [hunk 2]
```diff
@@ -2501,6 +2501,11 @@
         }
       }
     },
+    "css-unit-converter": {
```
Comment (rahilb) [inline package-lock.json]: is this actually used in this PR ?

Comment (walaura) [reply package-lock.json]: package-lock is a bit of a mess it should have been commited before

**Pull request merged**